docs/examples/viz_billboard_labels_new_font.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+"""
+=================================
+Billboard labels new font example
+=================================
+
+If you want to see the labels with a different font, you need to install
+the Freetype library and freetype-py (`pip install freetype-py`).
+
+"""
+
+###############################################################################
+# First, let's import some useful functions
+import fury
+from fury import actor, window
+from fury import text_tools
+import numpy as np
+
+# Set to True to enable user interaction
+interactive = True
+
+# Create a window FURY
+scene = window.Scene()
+
+
+###############################################################################
+# To create a label with a different font, you need to have the FreeType
+# library and freetype-py installed.
+#
+if text_tools._FREETYPE_AVAILABLE:
+
+    # We start chosing a path for the TTF file. Here we use the Roboto font
+    # that is available on the FURY examples folder.
+    font_path = font_path = f'{fury.__path__[0]}/data/files/RobotoMonoBold700.ttf'
```

Comment:
- Why twice `font_path`?
- `fury.__path__[0]` there is an existing variable instead of this I think. To check-in `fury.data` or `fetcher`